hello/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(int argc, char *argv[]){
+    std::cout << "Hello, world!" << std::endl;
```

Comment:
How about being a bit more welcoming?

```suggestion
    std::cout << "Hello, dear world!" << std::endl;
```